File: 7KYU/Evens times last/solution.js
function evenLast(numbers) {
    if(numbers.length == 0) return 0;
    
    var total = 0;
    
    for(var i = 0; i < numbers.length; i+=2) {
      total += numbers[i];
    }
    
    return total * numbers[numbers.length-1];
}

File: 7KYU/Evens times last/solution.c
#include <stddef.h>

int even_last(const int arr[], size_t z) {
  int total = 0;
  
  if(z == 0) return 0;
  
  for(int i = 0; i < z; i+=2) {
    total += arr[i];
  }
  return total * arr[z-1];
}